### DISCUSSION TITLE
Hey! Is there something like repomap in this plugin?

### DISCUSSION THREAD

**Discussion by antosha417** [olimorris/codecompanion.nvim#393] [Ideas]
I just saw that [avante.nvim](https://github.com/yetone/avante.nvim/pull/788) uses treesitter to build [repomap](https://aider.chat/docs/repomap.html) for `@codebase` command. 
What do you think about doing something similar? I bet repomap will help new `full_stack_dev` to be even more awesome

**Comment by olimorris** [comment]
You better believe it 😄.

The whole tool/agent logic is in a much better place as of `v9.0.0` and a `@workspace` tool is like the third bullet point down in my roadmap/todolist for this version. Given Aider has most of the Tree-sitter queries ready to go then I'm hoping it will be pretty straightforward (albeit I'll need to add a mechanism for people to manage tokens).
